<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1LV8K5I" failures="2" tests="21" name="Test" time="398.828" errors="0" timestamp="2021-03-25T20:08:11 IST">
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_001" time="9.098"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_002" time="16.024"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_003" time="18.868"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_004" time="23.671"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_005" time="13.297"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_006" time="27.339"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_007" time="24.128"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_008" time="108.493"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_009" time="17.634"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_010" time="5.923"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_011" time="13.139"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_012" time="12.69"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_013" time="9.889"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_014" time="9.895"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_015" time="10.952"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_016" time="11.611"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_017" time="18.973"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_018" time="16.408"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_019" time="9.001"/>
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_020" time="10.273">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.practo.tests.TestCases.TC_FH_020(TestCases.java:724)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC_FH_020 -->
  <testcase classname="com.practo.tests.TestCases" name="TC_FH_021" time="11.296">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.practo.tests.TestCases.TC_FH_021(TestCases.java:754)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC_FH_021 -->
</testsuite> <!-- Test -->
